backend/src/agents/config/tasks.yaml

```
@@ -0,0 +1,105 @@
+        {
+        "status": "SUCCESS|FAILURE|PARTIAL_FAILURE",
+        "step_description": "brief description of attempted step",
+        "result_summary": "concise summary of what happened",
+        "error_details": "detailed explanation if status is not SUCCESS",
+        "suggestions_for_planner": "recommendations for next step adaptation",
+        "outputs_created": ["list of any files/results produced"],
+        "next_step_context": "any important context for the next step"
+        }
+  
+    validation_criteria:
+        - "Single step execution attempted"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)